src/Umbraco.Web/HealthCheck/NotificationMethods/NotificationMethodBase.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Umbraco.Core.Configuration.HealthChecks;
+
+namespace Umbraco.Web.HealthCheck.NotificationMethods
+{
+    public abstract class NotificationMethodBase
+    {
+        protected NotificationMethodBase(bool enabled, bool failureOnly, HealthCheckNotificationVerbosity verbosity)
+        {
+            Enabled = enabled;
+            FailureOnly = failureOnly;
+            Verbosity = verbosity;
+        }
+
+        public bool Enabled { get; set; }
```

Comment:
Should these really be get/set since they are set in the ctor? Shouldn't they be readonly then?